test/basic.js

```diff
@@ -228,6 +228,31 @@ t.test('exec execPath', function (t) {
     })
   })
 
+  t.test('spaces in path on windows', { skip: !IS_WINDOWS }, function(t) {
```

Comment:
I don't really have access to Windows, can you confirm that this test fails without the change in `lib/mungers/cmd.js`?